docs/providers/aws/guide/iam.md

```diff
@@ -27,11 +27,11 @@ provider:
       name: custom-role-name
       path: /custom-role-path/
       statements:
-        - Effect: 'Allow',
-          Resource: '*',
-          NotAction: 'iam:DeleteUser',
+        - Effect: 'Allow'
+          Resource: '*'
+          Action: 'iam:DeleteUser'
```

Comment:
Note: I've also changed `NotAction` -> `Action`, this is a much more familiar and common use case.

`NotAction` can be confusing, and an easy typo to make when copy-pasting the example from the docs. For example myself I copy-pasted the example and later realized that it was `NotAction` (also first time I see this).